packages/legend-query-builder/style/_query-builder-olap.scss

```diff
@@ -0,0 +1,408 @@
+
+.query-builder__olap {
+  &__columns {
+    padding: 0.5rem 0;
+  }
+
+  &__column {
+    @include flexVCenter;
+
+    color: var(--color-light-grey-400);
+    padding: 0 0.5rem;
+
+    &:hover {
+      background: var(--color-dark-blue-shade-100);
+    }
+  }
+
+  &__column__context-menu {
+    display: flex;
+    height: 100%;
+    width: 100%;
+  }
+
+  &__column--selected-from-context-menu {
+    background: var(--color-dark-blue-shade-100);
+  }
+
+  &__column__drag-handle {
+    @include flexCenter;
+
+    background: var(--color-dark-grey-100);
+    height: 2.4rem;
+    width: 1.6rem;
+    min-width: 1.6rem;
+    margin-left: 0.5rem;
+
+    svg {
+      color: var(--color-dark-grey-300);
+      font-size: 1.2rem;
+    }
+
+    &__container {
+      @include flexCenter;
+
+      height: 3.4rem;
+    }
+  }
+
+  &__column__name {
+    @include flexVCenter;
+    margin-left: 0 0.5rem 0 0.5rem;
+    height: 3.4rem;
+    width: calc(100% - 15rem);
+    min-width: 10rem;
+
+    &__input {
+      padding: 0.5rem;
+      height: 2.8rem;
+      line-height: 2.8rem;
+      background: var(--color-dark-grey-100);
+      border: 0.1rem solid var(--color-dark-grey-300);
+      color: var(--color-light-grey-200);
+
+      &:focus {
+        border: 0.1rem solid var(--color-blue-200);
+        outline: none;
+      }
+    }
+  }
+
+  &__column__operation {
+    @include flexVCenter;
+
+    padding: 0 0.5rem;
+    height: 3.4rem;
+
+    &__operator {
+      @include flexVCenter;
+
+      height: 2.8rem;
+    }
+
+    &__operator__label {
+      @include flexCenter;
+
+      height: 100%;
+      padding: 0 1rem;
+      font-size: 1.2rem;
+      font-weight: 500;
+      background: var(--color-dark-grey-280);
+      color: var(--color-light-grey-100);
+      border-radius: 0.2rem 0 0 0.2rem;
+      white-space: nowrap;
+      width: 4rem;
+
+      &__agg {
+        width: 14rem;
+      }
+    }
+
+    &__operator__dropdown {
+      @include flexVCenter;
+
+      height: 100%;
+    }
+
+    &__operator__badge {
+      @include flexCenter;
+
+      height: 100%;
+      width: 2.8rem;
+      background: var(--color-yellow-200);
+      color: var(--color-light-grey-100);
+      border-radius: 0.2rem 0 0 0.2rem;
+
+      svg {
+        color: var(--color-dark-grey-280);
+      }
+    }
+
+    &__operator__dropdown__option {
+      font-size: 1.1rem;
+      height: 2.8rem;
+      font-family: 'Roboto Mono', monospace;
+    }
+
+    &__operator__dropdown__trigger {
+      @include flexCenter;
+
+      height: 100%;
+      width: 1.8rem;
+      background: var(--color-dark-grey-200);
+      border-radius: 0 0.2rem 0.2rem 0;
+
+      svg {
+        font-size: 1.2rem;
+        color: var(--color-light-grey-100);
+      }
+    }
+  }
+
+  &__column__actions {
+    @include flexVCenter;
+
+    height: 100%;
+    width: 3.4rem;
+  }
+
+  &__column__action {
+    @include flexCenter;
+
+    height: 3.4rem;
+    width: 3.4rem;
+
+    .query-builder__icon,
```

Comment:
this looks weird, do you mean to do `.query-builder__icon` only, why do we need the `svg`?